tools/FakeItEasy.Deploy/FakeItEasy.Deploy.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Octokit" Version="$(OctokitVersion)" />
+    <PackageReference Include="SimpleExec" Version="2.3.0" />

```

Comment:
Suck this version up as well and use the variable here and in FakeItEasy.Build?

Reply:
OK